client/gulpfile.js

```diff
@@ -246,10 +245,11 @@ async function installDependenciesFromXML(xmlPath, pluginDir) {
                     );
 
                     if (installResult.status === 0) {
-                        await fs.copy(
-                            path.join(pluginDir, "node_modules", pkgName, "static"),
-                            path.join(pluginDir, "static")
-                        );
+                        const packagePath = path.join(pluginDir, "node_modules", pkgName);
+                        const defaultPath = path.join(packagePath, "static");
+                        const alternatePath = path.join(packagePath, "dist/static");
+                        const sourcePath = fs.existsSync(defaultPath) ? defaultPath : alternatePath;
```

Comment:
Instead of adding extra directories that get staged I would really prefer adjusting the published package for ngl viz -- I can PR on that side so we can keep this list minimal.  The more we can limit and/or eliminate "alternate paths", etc., the more robust all this becomes and stays.

I can make a PR on the other side for this and update this PR after.

Reply:
That works. Thanks!